disco/types/user.py

```diff
@@ -22,16 +22,13 @@ class User(SlottedModel, with_equality('id'), with_hash('id')):
 
     presence = Field(None)
 
-    def get_avatar_url(self, fmt='webp', size=1024):
```

Comment:
Format shouldn't be removed; you can still request static versions of GIF avatars. I also don't know why you would remove `size`?

Ideally you should accept whatever format the user requests, if not default to `webp`/`gif`. I imagine you would do this by defaulting the arg to `None`, on which you do format detection, otherwise you accept the passed format.

Reply:
Patched in commit `87172ff`.